src/HotChocolate/Core/src/Types.Scalars/ScalarResources.Designer.cs

```diff
@@ -11,32 +12,46 @@ namespace HotChocolate.Types.Scalars {
     using System;
     
     
-    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
-    [System.Diagnostics.DebuggerNonUserCodeAttribute()]
-    [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
+    /// <summary>
+    ///   A strongly-typed resource class, for looking up localized strings, etc.
+    /// </summary>
+    // This class was auto-generated by the StronglyTypedResourceBuilder
+    // class via a tool like ResGen or Visual Studio.
+    // To add or remove a member, edit your .ResX file then rerun ResGen
+    // with the /str option, or rebuild your VS project.
```

Comment:
Interesting. Are you using Visual Studio? 
@michaelstaib  how do we handle different formats of the resources depending on the IDE?

Reply:
I will have a look ....

Reply:
This looks alright.

Its generated ... so as long as it works we ignore this code.

Reply:
[global::System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
 
 sonar will ship code that is generated